modules/olm-fb-catalogs-schemas.adoc

```diff
@@ -0,0 +1,158 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm/olm-packaging-format.adoc
+
+[id="olm-fb-catalogs-schemas_{context}"]
+= Schemas
+
+File-based catalogs use a format, based on the link:https://cuelang.org/docs/references/spec/[CUE language specification], that can be extended with arbitrary schemas. The following  `_Meta` CUE schema defines the format that all file-based catalog blobs must adhere to:
+
+.`_Meta` schema
+[source,go]
+----
+_Meta: {
+  // schema is required and must be a non-empty string
+  schema: string & !=""
+
+  // package is optional, but if it's defined, it must be a non-empty string
+  package?: string & !=""
+
+  // properties is optional, but if it's defined, it must be a list of 0 or more properties
+  properties?: [... #Property]
+}
+
+#Property: {
+  // type is required
+  type: string & !=""
+
+  // value is required, and it must not be null
+  value: !=null
+}
+----
+
+[NOTE]
+====
+No CUE schemas listed in this specification should be considered exhaustive. The `opm validate` command has additional validations that are difficult or impossible to express concisely in CUE.
+====
+
+An Operator Lifecycle Manager (OLM) index currently uses three schemas (`olm.package`, `olm.channel`, and `olm.bundle`), which correspond to OLM's existing package and bundle concepts.
```

Comment:
I'd suggest trying to avoid the parentheses here, but I'm struggling a bit to find a better solution other than: 

An Operator Lifecycle Manager (OLM) index currently uses three schemas: `olm.package`, `olm.channel`, and `olm.bundle`, which correspond to OLM's existing package and bundle concepts.